elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -37,18 +38,23 @@ async def verify_attached_bugs_cli(runtime: Runtime, verify_bug_status: bool, ad
         exit(1)
     if runtime.use_jira:
         await verify_attached_bugs(runtime, verify_bug_status, advisories, verify_flaws, True)
-    await verify_attached_bugs(runtime, verify_bug_status, advisories, verify_flaws, False)
+    else:
+        await verify_attached_bugs(runtime, verify_bug_status, advisories, verify_flaws, False)
 
 
 async def verify_attached_bugs(runtime: Runtime, verify_bug_status: bool, advisories: Tuple[int, ...], verify_flaws: bool, use_jira: bool):
     validator = BugValidator(runtime, use_jira, output="text")
     try:
         await validator.errata_api.login()
-        advisory_bugs = await validator.get_attached_bugs(advisories)
-        non_flaw_bugs = validator.filter_bugs_by_product({b for bugs in advisory_bugs.values() for b in bugs})
+        advisory_bug_map = validator.get_attached_bugs(advisories)
```

Comment:
any reason await disappeared here?

Reply:
Yes earlier we were doing async calls in get_attached_bugs but now we are not. I'll explain my reasoning as to why in the method